packages/manager/apps/dedicated/client/app/dedicated/server/servers.html

```diff
@@ -21,7 +21,7 @@
             data-sortable
             data-type="string"
         >
-            <a data-ng-href="{{ $ctrl.getServerDashboardLink($row) }}"
+            <a href="" data-ng-click="$ctrl.goToServerDashboard($row)"
```

Comment:
maybe change it for a button if href is empty

Reply:
I change with a oui-button, I found a "link" variant of button but it is display as ghost variant.